source/content/drupal-9.md

```diff
@@ -0,0 +1,161 @@
+  terminus multidev:create <site>.<env> update-9.0.0-beta3
+  ```
+
+1. Create and switch to a new testing branch in the D8 codebase:
+
+  ```bash{promptUser: user}
+  git checkout -b update-9.0.0-beta3
+  ```
+
+1. Pull the `update-9.0.0-beta3` branch into your codebase:
+
+  ```bash{promptUser: user}
+  git pull https://github.com/pantheon-systems/drops-8.git update-9.0.0-beta3
+  ```
+
+1. Modify the `pantheon.yml` file to specify PHP 7.4 and Drush 8:
+
+  ```yaml:title=pantheon.yml
+  php_version: 7.4
+  drush_version: 8
+  ```
+
+1. Run `update.php` on your new D9 site:
+
+  ```bash{promptUser: user}
+  terminus drush <site>.<env> updatedb
+  ```
+
+## Test an Existing Drupal Site for D9 Upgrade Compatibility
+
+1. Review the [How to Prepare Your D7 or D8 Site for D9](https://www.drupal.org/docs/9/how-to-prepare-your-drupal-7-or-8-site-for-drupal-9) guide on Drupal.org.
+1. Use the [Upgrade Status Drupal 8/9 module](https://www.drupal.org/project/upgrade_status) to generate a full report of your site’s compatibility.
+1. Check out [Acquia’s D9 Deprecation Status Upgrade Tracker](https://dev.acquia.com/drupal9/deprecation_status) for information about D9 support for contributed Drupal modules and themes.
+
+### Update Deprecated Code for D9 Compatibility
+
+D9 has deprecated a number of different functions and APIs in favor of better options going forward.
+
+For example, `node_load()` was replaced in D9 with `Node::load` resulting in this change needing to be made:
+
+D8:
+
+```none
+$node = node_load(1);
+```
+
+D9:
+
+```none
+use \Drupal\node\Entity\Node;
+$node = Node::load(1);
+```
+
+Since most of these changes are relatively minor, there are a number of [deprecation checking and correction tools](https://www.drupal.org/docs/9/how-to-prepare-your-drupal-7-or-8-site-for-drupal-9/deprecation-checking-and-correction-tools) available.
+
+## Test Upgrade an Existing Pantheon D8 Site to D9
+
+Already running a Pantheon site using our [Drupal 8 upstream](https://github.com/pantheon-systems/drops-8)? Want to spin up one real quick to test it out? Leverage the amazing power of our Multidev tooling to test D9 in a git branch.
+
+1. Clone your D8 [site’s codebase to your computer](https://pantheon.io/docs/local-development#get-the-code). You can create a new D8 site or use an existing D8 site:
+
+  ```bash{promptUser: user}
+  git clone <url for site repo>
+  ```
+
+1. Use Terminus to create a [Multidev](/multidev) environment called `d9-beta` on your D8 site for testing:
+
+  ```bash{promptUser: user}
+  terminus multidev:create <site>.<env> d9-beta
+  ```
+
+1. Create and switch to a new testing branch in the D8 codebase:
+
+  ```bash{promptUser: user}
+  git checkout -b d9-beta
+  ```
+
+1. Pull the `drupal9-beta3-early-access-not-for-production-use` branch into your codebase:
+
+  ```bash{promptUser: user}
+  git pull https://github.com/pantheon-systems/drops-8.git drupal9-beta3-early-access-not-for-production-use
```

Comment:
I think this tag was made with a commit on top of 8.8.5. So for this part to work cleanly I had to be on a git history with Drupal 8.8.5. When I tried this when on a site on 8.8.6 I got a huge merge conflict. I thought that adding `-X theirs` would work but it did not. @greg-1-anderson any tips?

Reply:
Yes, this is how it was expected to work (or not work, as the case may be).

We could rewrite the 9.0.0-beta3 tag on top of the 8.8.6 release, if desired.

Reply:
Yes, let's make another one. How about the name `drupal9-beta3--from-8.8.6--early-access-not-for-production-use`